### PR TITLE
Add tooltips with keyboard shortcuts to playback controls

### PR DESCRIPTION
**Pull request by 2metres**
**User-Facing Changes**
Add tooltips to Playback controls.

### CODE REVIEW

**Comment by defunctzombie** [comment]
We need to profile this to make sure this thing is not rendering at 60fps - similar to the 3d panel where we convinced ourselves this is ok. Alternatively find a way to make the tooltip component use less scary. We need to be mindful of the impact MUI's lack of performance care has on our app.

**Comment by 2metres** [comment]
From my observation I think the Play Loop and Seek buttons are safe. 

Unrelated to the tooltip though
I am seeing an issue when the loop playback option is active where the play button changes to the pause momentarily at when the loop start/end happens. Which causes the component to rerender as the icon has changed

**Comment by 2metres** [comment]
Can you direct me to the issue where we determined that the MUI tooltip has performance issues? So that I can better understand the cases where I **has** caused issues.

**Comment by amacneil** [comment]
:+1:

File: packages/studio-base/src/components/PlaybackControls/index.tsx [hunk 1]
```diff
@@ -221,43 +221,67 @@ export default function PlaybackControls(props: {
...
+                size="small"
+                onClick={togglePlayPause}
+                icon={isPlaying ? <Pause20Regular /> : <Play20Regular />}
+                activeIcon={isPlaying ? <Pause20Filled /> : <Play20Filled />}
+              />
+            </Tooltip>
+            <Tooltip
+              title={
+                <>
+                  Seek foreward <kbd>→</kbd>
```
Comment (amacneil) [inline packages/studio-base/src/components/PlaybackControls/index.tsx:260]:
```suggestion
                  Seek forward <kbd>→</kbd>
```

File: packages/theme/src/components/MuiTooltip.ts [hunk 2]
```diff
@@ -21,6 +23,14 @@ export const MuiTooltip: OverrideComponentReturn<"MuiTooltip"> = {
       backdropFilter: "blur(3px)",
       fontWeight: "normal",
       fontSize: theme.typography.caption.fontSize,
+
+      kbd: {
+        background: tc(theme.palette.common.white).darken(45).toString(),
```
Comment (defunctzombie) [inline packages/theme/src/components/MuiTooltip.ts:28]: Does mui not have color utilities we can use to darken/lighten colors?

Comment (2metres) [reply packages/theme/src/components/MuiTooltip.ts:28]: It does but we use tinycolor2 elsewehere as well. For color manipulation I'm trying to stick with tinycolor2

Comment (defunctzombie) [reply packages/theme/src/components/MuiTooltip.ts:28]: Why?

File: packages/theme/src/components/MuiTooltip.ts [hunk 3]
```diff
@@ -2,25 +2,35 @@
...
 export const MuiTooltip: OverrideComponentReturn<"MuiTooltip"> = {
   defaultProps: {
     arrow: true,
     TransitionComponent: Fade,
+    enterDelay: 500,
   },
   styleOverrides: {
     arrow: ({ theme }) => ({
-      color: alpha(theme.palette.grey[700], 0.92),
+      color: tc(theme.palette.grey[700]).setAlpha(0.92).toString(),
```
[the remark this answers is not in the document]
Comment (2metres) [reply packages/theme/src/components/MuiTooltip.ts:18]: I was recently asked a question about which color manipulation library we are using consistently across the app. Noticed that this was using the MUI alpha function and moved to use tinycolor2 instead

**Comment by 2metres** [comment]
Closing in favor of #7364

**Pull request closed without merging**